### ISSUE TITLE
Suppport inquirer default

### ISSUE THREAD

**Issue by pauldijou** [yeoman/yeoman-app#51]
Each inquirer prompt can have a default value among the possible choices. It would be nice if the correspond HTML input was already selected accordingly.

**Comment by ruyadorno** [comment]
This should be the case already, @pauldijou have you come across any situation where the `default` value is not working?

**Comment by pauldijou** [comment]
True. Maybe it's related to #43 then since I only have the problem for radio buttons that I cannot select.
